LocalWiki.php

```diff
@@ -44,6 +44,24 @@ function onBeforePageDisplay( OutputPage $out ) {
 
 		break;
 	case 'gratisdatawiki':
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
+		
+		function onBeforePageDisplay( OutputPage $outputPage ) {
+			$outputPage->addMeta( 'og:image:width', '1200' );
+			
+			if ( isset( $meta['title'] ) ) {
+				$outputPage->addMeta( 'og:title', $meta['title'] );
+			}
+			
+			if ( isset( $meta['description'] ) ) {
```

Comment:
this won't do anything, as $meta is never set anywhere within this hook, so the `if` always returns false, and nothing is executed/added here.